applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -165,6 +167,3 @@ def Initialize(self):
 
         (self.solver).SetEchoLevel(self.settings["echo_level"].GetInt())
         self.solver.Initialize()
-
-    def AdvanceInTime(self, current_time):
```

Comment:
Do you need to remove this to make your workflow work?

Reply:
It wasn't used since we do not call for AdvanceinTime() in CoSimulationSteadyAnalysis